src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+        case ("event"):
+            if (inputArr.length < 2) {
+                throw new IllegalTaskException();
+            } else if (!input.contains("/")) {
+                throw new DueDateFormatException();
+            }
+            addEvent(input, user);
+            System.out.println("\t" + addTaskSalutation(input.substring(6, input.indexOf("/") - 1)) + "\n\tNow you have " + user.listLength + " tasks in the list, " + user.tasksIncomplete + " incomplete tasks.");
+            break;
+        case ("done"):
+            System.out.println(markDone(input, user));
+            break;
+        case ("delete"):
+            if (inputArr.length != 2) {
+                throw new DeleteTaskFormatException();
+            }
+            deleteEvent(inputArr[1], user);
+            break;
+        default:
+            throw new IllegalCommandException();
+        }
+        writeToFile(user);
+    }
+    public static void greet(String logo) {
+        System.out.println("\n" + logo);
+        line();
+        System.out.println("\tHello Sir,\n" + "\tWhat can I do for you?\n"); // greet
+        line();
+    }
+    public static void exit() {
+        System.out.println("Goodbye Sir. I shall be at your service again whenever you require.\n"); //exit message
+        line();
+    }
+    public static void openSaveFile (Tasks user) {
+        try {
+            File directory = new File ("data");
+            if (!directory.exists()) {
+                directory.mkdir();
+            }
+            save = new File(directory + "/" + "duke.txt");
+            save.createNewFile();
+            Scanner s = new Scanner(save);
+            int i = 1;
+            while (s.hasNext()) {
+                String[] parts = s.nextLine().split("\\|");
+                String type = parts[0].trim();
+                String completion = parts[1].trim();
+                String item = parts[2].trim();
+                switch (type) {
+                    case "T":
+                        addTodo("todo " + item, user);
+                        break;
+                    case "D":
+                        addDeadline("deadline " + item, user);
+                        break;
+                    case "E":
+                        addEvent("event " + item, user);
+                        break;
+                }
+                if (completion.equals("X")) {
+                    markDone("done " + Integer.toString(i), user);
+                }
+                i++;
+            }
+        } catch (IOException | MultiMarkDoneException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+    private static void writeToFile(Tasks user) throws IOException {
+        FileWriter fw = new FileWriter(save);
+        for (int i = 0; i < user.listLength; i++) {
+            fw.write(user.list[i].getType() + " | " + user.list[i].getStatus() + " | " + user.list[i].getOriginalInput() +  System.lineSeparator());
```

Comment:
This line is rather long, and could trip up the reader. The guideline is to have [<120 chars](https://se-education.org/guides/conventions/java/basic.html#layout) per line. 

Refer to the[ java string format() method ](https://www.javatpoint.com/java-string-format)example for another way to format long strings.